LICENSE.md

```diff
@@ -67,58 +67,59 @@ The User may not limit access to the Mod in any way, except as provided by the G
 The User may decompile compiled binaries of the Mod.  
 The User may examine the Mod's source code.
 
-5. Right to distribute
+5. No right to distribute
 --------
 
-The Author reserves the right to distribute the Mod.  
-The Author may designate a person or organization as a Distributor.  
-A Distributor is granted the right to distribute the Mod.  
-The Author may require that a Distributor provides a direct link to the Codebase.  
-The User may not distribute the Mod without being granted distribution rights from the Author.  
-The Author may revoke distribution rights from any Distributor.  
+The Author reserves the right to transmit verbatim copies of ("Distribute") the Mod.  
```

Comment:
I can see that, but "transmit" != "distribute". I can distribute by handing someone a flash drive with the mod on it, or burning CDs and then handing them out. 

Basically, transmit is the wrong word, and distribute is using the dictionary definition...we don't need to define every word we use, that would be madness...just the ones with specific meanings that may not match the exact dictionary definition.

I could be wrong on this...but that's why I'm going to have a lawyer look at it before I finalize it.